src/Components/Assets/AssetsList.tsx

```diff
@@ -240,50 +240,55 @@ const AssetsList = () => {
     manageAssets = (
       <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 md:-mx-8 gap-2">
         {assets.map((asset: AssetData) => (
-          <div
-            key={asset.id}
-            className="w-full bg-white rounded-lg cursor-pointer border-1 shadow p-5 justify-center items-center border border-transparent hover:border-primary-500"
-            onClick={() =>
-              navigate(
-                `facility/${asset?.location_object.facility.id}/assets/${asset.id}`
-              )
-            }
+          <Link
+            href={`/facility/${asset?.location_object.facility.id}/assets/${asset.id}`}
+            style={{ textDecoration: "none", color: "inherit" }}
           >
-            <div className="md:flex">
-              <p className="text-xl flex font-medium capitalize break-words">
-                <span className="mr-2 text-primary-500">
-                  <CareIcon
-                    className={`care-l-${
-                      (
-                        (asset.asset_class &&
-                          assetClassProps[asset.asset_class]) ||
-                        assetClassProps.NONE
-                      ).icon
-                    } text-2xl`}
-                  />
+            <div
+              key={asset.id}
+              className="w-full bg-white rounded-lg cursor-pointer border-1 shadow p-5 justify-center items-center border border-transparent hover:border-primary-500"
+              onClick={() =>
+                navigate(
+                  `facility/${asset?.location_object.facility.id}/assets/${asset.id}`
+                )
+              }
```

Comment:
@souravghosh01 do we need the onClick to navigate too when the `Link` is already serving the purpose?
```suggestion
```

Reply:
@rithviknishad Thank you for noticing. We don't need the `onClick` It can be removed.